package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "@types/react-redux": "^7.1.20",
+    "aos": "^2.3.4",
```

Comment:
revert changes for _package.json_ and _package-lock.json_ files.
This [https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha](link) can be useful